portfolio.html

```diff
@@ -0,0 +1,120 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link href="portstyle.css" type="text/css" rel="stylesheet">
+  
+</head>
+<body>
+
+	<section class="navigationBar">
```

Comment:
I'd probably use the nav tag here to give your structure more specificity.

Reply:
Good idea have ammended.